### DISCUSSION TITLE
✅ Add post_type selector in Search Results conditions

### DISCUSSION THREAD

**Discussion by FireDeveloper** [elementor/elementor#21299] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I have multiple post types.
- products
- posts
- guides
- news
- job positions

Currently there is only separation between product search and everything else by using different site part on theme builder.
Product archive for products and archive for everything else.

I have different archive page for guides, different archive for news, different archive for job positions

I want to use individual search result page for each archive as it currently works on product archive.
By using the attributes `name="post_type" value="product"` it is possible to target only the products and display the product archive page but for every other post type there is the option only for a global search result page.

### Describe the solution you'd like

On display condition for the option `Search Results` add the option to display by query (i.e. products, posts, guides, news etc)

**Comment by nicholaszein** [comment]
Hello there @FireDeveloper!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.23! 🥳 

You can now use the new [Search widget with Live Results](https://elementor.com/help/search-widget/), that includes a query source to define which posts to query.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
